projects/packages/classic-theme-helper/.phan/baseline.php

```diff
@@ -9,8 +9,8 @@
  */
 return [
     // # Issue statistics:
-    // PhanUndeclaredClassMethod : 20+ occurrences
```

Comment:
Had to made this change due to [this job failure](https://github.com/Automattic/jetpack/actions/runs/14649989803/job/41113205300). Not sure how it is related to my changes.